src/index.js

```diff
@@ -30,6 +30,8 @@ module.exports = ({ Store }) => {
         intervalMs:
           (options.expired && options.expired.intervalMs) ||
           clearExpiredInterval,
+        unrefInterval: (options.expired && options.expired.unrefInterval) ||
```

Comment:
we'd have to add any new option property to the documentation. And why do we need "|| false" ?

Reply:
Yes, I will add it to the docs if you accept it to be added. Regarding the "false", that's redundant of course and I'll remove it.